packages/main/src/ColorValue.ts

```diff
@@ -0,0 +1,171 @@
+import {
+	getRGBColor,
+	HSLToRGB,
+	HEXToRGB,
+	RGBToHSL,
+	RGBtoHEX,
+} from "@ui5/webcomponents-base/dist/util/ColorConversion.js";
+import type {
+	ColorHSL,
+	ColorRGB,
+} from "@ui5/webcomponents-base/dist/util/ColorConversion.js";
+
+/**
+ * @class
+ *
+ * ### Overview
+ *
+ * The `ui5-color` class represents the `value` used in `ui5-color-picker`
+ *
+ * A color can be represented using rgb, hsl, or hex values. A color also has an alpha value.
+ * @public
```

Comment:
Shouldn't we document this as a private class as it's a helper class used only internally. Also the tag name `ui5-color` is not specified anywhere as this class doesn't have a template.